instances/supportdocs/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2.4'
+
+networks:
+  web:
+    external: true
+
+services:
+  jekyll:
+    image: bretfisher/jekyll-serve
```

Comment:
We can't use this live as it's designed for local development and also doesn't have any simple way to override parameters since they're [hard-coded](https://github.com/BretFisher/jekyll-serve/blob/master/Dockerfile.serve#L11).